sct.py

```diff
@@ -106,9 +106,9 @@ def clean_resources(ctx, user, test_id, logdir, config_file, backend):  # pylint
 
         for _test_id in test_id:
             params['TestId'] = _test_id
-            if backend == 'aws':
+            if 'aws' in backend:
```

Comment:
How about `backend in ['aws', 'aws-siren']`? it's more strict

Reply:
we can assume this param is in the limit list we define, i.e. we check it before the run even start.
and this is relvent to everything aws related, this is safe enough, from my POV.

and this code doesn't have unittests, hence we won't notice if there would be a new type of AWS backend.